distribution/lib/Standard/Base/0.0.0-dev/src/Data/Vector.enso

```diff
@@ -91,41 +98,27 @@ new_builder (capacity=10) = Builder.new capacity
    A vector allows to store an arbitrary number of elements in linear memory. It
    is the recommended data structure for most applications.
 from_polyglot_array : Any -> Vector Any
-from_polyglot_array arr = Vector_Data (Proxy_Polyglot_Array.Proxy_Polyglot_Array_Data arr)
-
-## The basic, immutable, vector type.
-type Vector a
-
-    ## ADVANCED
+from_polyglot_array array = @Builtin_Method "Vector.from_polyglot_array"
```

Comment:
I don't like the naming convention between this and `from_array`. The difference between them is one copies the array, the other does not. The are both equally capable of working on polyglot arrays. I'd suggest something like `from_array` and `unsafe_from_array`, with good documentation when to use which.

Reply:
Can we go with a single method?
```
from_array :: Any -> Boolean -> Vector
from_array array trust=False = ...
```

Reply:
This is just subjective opinion, but I'd prefer to have two separate methods in the public API (they can use some common private method under the hood) - so that it is immediately clear where the 'unsafe' one is used - and with the separate method we can even just grep, whereas with parameter it becomes more complicated. Although this is just a stylistic preference.

If we go for a single method, I'd suggest a different name for the parameter - `trust` does not convey the right meaning in my opinion. Maybe sth along the lines of `unsafe_skip_copy`?

Reply:
I've reported [Consolidate Vector & Array](https://www.pivotaltracker.com/n/projects/2539304/stories/183217534) story to resolve the name issue & list of available methods on both types.